okio/src/main/kotlin/okio/Util.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("Util")
+
+package okio
+
+internal fun checkOffsetAndCount(size: Long, offset: Long, byteCount: Long) {
```

Comment:
Does defining these in a `Util` object avoid the japicmp problem? I’d love to get rid of the last `.java` files here.

Reply:
It doesn't, you'll still have a `public class Util`